### ISSUE TITLE
Populate Manufacturer field in downloaded SBOM's

### ISSUE THREAD

**Issue by thompson-tomo** [DependencyTrack/dependency-track#3935]
### Current Behavior

Property is not included

### Proposed Behavior

When a SBOM is generated by Dependency track the manufacturer property as defined in (https://github.com/CycloneDX/specification/blob/62a669075f1897193a14060e0784e6a7576b693d/schema/bom-1.6.schema.json#L677) should be populated with information about OWASP etc.

### Checklist

- [X] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [X] I have checked the [existing issues](https://github.com/DependencyTrack/dependency-track/issues) for whether this enhancement was already requested

**Comment by PiyushVyas13** [comment]
Hi @andrewjmaguire . I would love to contribute to this. Can you please assign it to me?

**Comment by msymons** [comment]
@PiyushVyas13, I just want to check... are you wanting to work on this issue as a Hacktoberfest contribution?

**Comment by PiyushVyas13** [comment]
Yes. It is actually my first time contributing to an open source project, so I'm looking forward to working on this issue

**Comment by msymons** [comment]
@PiyushVyas13 , fFirst time contributing to OSS?   We are honored that you chose our project to be your first.

Welcome aboard, and have fun with Hackoberfest

**Comment by Zuplyx** [comment]
Populating the manufacturer is also required for compliance with the German [BSI TR-03183](https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Publications/TechGuidelines/TR03183/BSI-TR-03183-2_v2_1_0.pdf?__blob=publicationFile&v=5) Guideline. According to chapter 5.2.1 and the mapping in chapter 8.2 either the URL or the email has to be populated.
Therefore my recommendation would be to add these to the Identity tab in the project's details.

@PiyushVyas13 are you still working on this?
